packages/api/models/Environment.ts

```diff
@@ -1,6 +1,8 @@
+import { Application } from "../orm/entity/Application";
 import { AppEnvOperator } from "./Operator";
 
 export interface Environment {
   name: string;
-  operators: Array<AppEnvOperator>;
+  application: Application;
```

Comment:
Why do you have application here?
It is not required on the internal API yaml spec.

Reply:
Good point. I was probably thinking of it from the database side (which includes application) rather than the spec side. I did run into a conflict when trying to create an environment and `Application` was needed for a simple interface of `name` and `application`. I'll update accordingly.